src/annotator/integrations/html.js

```diff
@@ -0,0 +1,46 @@
+import { anchor, describe } from '../anchoring/html';
+
+import { HTMLMetadata } from './html-metadata';
+
+/**
+ * @typedef {import('../../types/annotator').Integration} Integration
+ */
+
+/**
+ * Document type integration for ordinary web pages.
+ *
+ * This integration is used for web pages and applications that are not handled
+ * by a more specific integration (eg. for PDFs).
+ *
+ * @implements {Integration}
```

Comment:
If you are not using `Integration` type only once, maybe you could do this:

```suggestion
 * @implements {import('../../types/annotator').Integration}
```